playbooks/roles/shadowsocks/vars/main.yml

```diff
@@ -30,6 +30,7 @@ shadowsocks_configure_flags: "--disable-documentation --prefix=/usr"
 
 shadowsocks_location: "/etc/shadowsocks-libev"
 shadowsocks_password_file: "{{ shadowsocks_location }}/shadowsocks-password.txt"
+shadowsocks_password_file_localpath: "../{{ streisand_local_directory }}/shadowsocks-password.txt"
```

Comment:
> If we decide to make the creation of this file subject to a conditional so it is only present for Vagrant runs, then this won't really apply

:+1:  - the fetch is conditional now.

>  if we plan on pulling down more local files we can do server-specific subdirectories too.

I think that's probably a good follow up issue.